src/thrift/jaeger.rs

```diff
@@ -118,10 +118,17 @@ pub struct SpanRef {
     pub trace_id_low: i64,
     pub trace_id_high: i64,
     pub span_id: i64,
+    pub parent_span_id: i64,
```

Comment:
Please remove this field because the associating thrift `struct` does not define it (see: https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift#L43).

Reply:
I followed the reference go implementation which uses it. For example it injects it here https://github.com/jaegertracing/jaeger-client-go/blob/master/propagation.go#L193
But I agree, that it currently does nothing in rust, so I removed it.